public/javascripts/help.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready( topicSelector() => {
+  $( 'a.topic-selector' ).click( ( event ) => {
+    event.preventDefault()
+
+    // const element = $( event.target)
```

Comment:
Comments - this functionality is not completed based on the comments; is this intended to be in another branch?